components/desktop/gst-plugins-good1/Makefile

```diff
@@ -0,0 +1,91 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Aurelien Larcher.  All rights reserved.
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME= gst-plugins-good1
+COMPONENT_VERSION= 1.10.3
+COMPONENT_SUMMARY= GNOME streaming media framework plugins
+COMPONENT_CLASSIFICATION= System/Multimedia Libraries
+COMPONENT_FMRI= library/audio/gstreamer1/plugin/good
+COMPONENT_SRC_NAME= gst-plugins-good
+COMPONENT_SRC= $(COMPONENT_SRC_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE= $(COMPONENT_SRC).tar.xz
+COMPONENT_ARCHIVE_HASH= \
+  sha256:4e07e93e34d4b93208f1579c21e7d91a236577b36f128a5332ffee85b4465955
+COMPONENT_ARCHIVE_URL= \
+  http://gstreamer.freedesktop.org/src/$(COMPONENT_SRC_NAME)/$(COMPONENT_ARCHIVE)
+COMPONENT_PROJECT_URL = http://gstreamer.freedesktop.org/
+COMPONENT_LICENSE= LGPLv2
+
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/configure.mk
+include $(WS_MAKE_RULES)/ips.mk
+
+COMPONENT_PREP_ACTION = ( cd $(SOURCE_DIR) && aclocal -I ./m4 -I./common/m4 &&\
+                          autoheader &&\
+                          automake -a -f -c --gnu &&\
+                          autoconf )
+gcc_OPT = -O2
```

Comment:
Is -O2 still necessary in these Makefiles? I've added it to gstreamer as I got really strange crashes with -O3 and some multimedia applications (now don't remember details).